base/src/test/resources/failure/tyck/positivity.aya.txt

```diff
@@ -0,0 +1,13 @@
+In file $FILE:4:4 ->
+
+  2 │   
+  3 │   open data Cat : Set
+  4 │     | meow : (Cat → Dead) → Cat
+    │       ╰──╯
+
+Error: Positivity check failed for | meow : (Cat → Dead) → Cat -> data Cat : Set
+       0
+         | meow : (Cat → Dead) → Cat
```

Comment:
I really don't understand this message....

Reply:
I agree, it's very messy.